erpnext/hr/report/vehicle_expenses/vehicle_expenses.js

```diff
@@ -22,18 +22,6 @@ frappe.require("assets/erpnext/js/financial_statements.js", function() {
 						query_report.trigger_refresh();
 					});
 				}
-			},
```

Comment:
fy = frappe.db.get_value('Fiscal Year', filters.get('fiscal_year'), ['year_start_date', 'year_end_date'], as_dict=True)
Fetched start date and end date from the fiscal year